lib/bpf/disk.go

```diff
@@ -74,10 +78,13 @@ type cgroupRegister interface {
 }
 
 type open struct {
-	session
+	//session
 
-	eventBuf *RingBuffer
-	lost     *Counter
+	objs diskObjects
+
+	eventBuf chan []byte
+	toClose  []io.Closer
+	//lost     *Counter
```

Comment:
@russjones Do you know if the BPF counters are used by anyone? From what I see, they are defined and initialized but not used. I want to remove them (including tests), but I'm curious if maybe you're aware of any history behind it. Is there any reason not to remove them?